vendor/github.com/containers/image/pkg/docker/config/config.go

```diff
@@ -64,11 +65,17 @@ func GetAuthentication(sys *types.SystemContext, registry string) (string, strin
 	}
 
 	dockerLegacyPath := filepath.Join(homedir.Get(), dockerLegacyCfg)
+	var paths []string
 	pathToAuth, err := getPathToAuth(sys)
-	if err != nil {
-		return "", "", err
+	if err == nil {
+		paths = append(paths, pathToAuth)
+	} else {
+		// Error means that the path set for XDG_RUNTIME_DIR does not exist
```

Comment:
This comment makes me a little uncomfortable. Can we confidently assert that this is the only possible way that `getPathToAuth()` can fail? Would it make sense to check the returned error and/or change the comment to 'Error _might_ mean that ...'?

Reply:
yup, that is the only error `getPathToAuth()` returns.